server/src/main/java/org/opensearch/telemetry/tracing/AttributeNames.java

```diff
@@ -94,4 +94,19 @@ private AttributeNames() {
      * Refresh Policy
      */
     public static final String REFRESH_POLICY = "refresh_policy";
+
+    /**
+     * Search Request Source
+     */
+    public static final String SOURCE = "source";
```

Comment:
The source could be huge, we should not make it part of the trace by default I think

Reply:
Removed it

Reply:
It is still there